packages/create-ponder/src/templates/basic.ts

```diff
@@ -1,17 +1,21 @@
-import { writeFileSync } from "node:fs";
-import path from "node:path";
-import prettier from "prettier";
+import { writeFileSync } from 'node:fs';
+import path from 'node:path';
+import prettier from 'prettier';
 
-import type { PartialConfig } from "@/index";
+import type { PartialConfig } from '@/index';
 
 export const fromBasic = ({ rootDir }: { rootDir: string }) => {
   const abiFileContents = `[]`;
 
-  const abiRelativePath = "./abis/ExampleContract.json";
+  const abiRelativePath = './abis/ExampleContract.json';
```

Comment:
Looks like you formatted (maybe on file save?) but it didn't use `.eslintrc`. No pressure to get this running (lmk if you'd rather me fix & ship this) but you might just need to `pnpm install` and then `pnpm lint:format` in the project root. Noticed that the `pnpm-lock.yaml` also has changes (it shouldn't).

Reply:
Happy for you to take the reins on this. Was having a weird issue where even after installing deps none of the scripts were running. Deleted it locally, but then realized its kinda hard to get back my branch if it wasn't something I pushed up against my local fork.

For future, to avoid `pnpm-lock.yaml` getting pushed, would you recommend just hand picking the files I've changed rather than `git add .`. And second question, regarding the changeset. Does it update to include the latest changes, after which for example, you've requested a review? Hope that makes sense. Thanks for your time!